### PR TITLE
Add security section to GitHub logins doc

### PR DESCRIPTION
**Pull request by thompiler**
Adds section explaining the new "require GitHub logins" feature:

screen shot 2017-05-10 at 1 52 52 pm

/cc @shifi

### CODE REVIEW

File: jekyll/_docs/integrations/github-logins.md [hunk 1]
```diff
@@ -21,3 +21,16 @@ just select the "Sign in with GitHub" button from the
...
+Airbrake allows you to require that all users in your Airbrake account use
+GitHub authentication to log in to Airbrake. When this setting is enabled, no
+users in your account will be able to log in using their email and password.
+Instead they will be required to use GitHub authentication to sign in.
+
+This option allows users to take advantage of
+[GitHub's two factor authentication](https://help.github.com/articles/about-two-factor-authentication/)
+to secure their Airbrake accounts.
+
+You can find this option in your [account settings](https://airbrake.io/account/edit)
```
Comment (shifi) [inline jekyll/_docs/integrations/github-logins.md]: `[account security settings](https://airbrake.io/account/security)` and then remove:
`under the "Security" tab.`

File: Gemfile.lock [hunk 2]
```diff
@@ -90,8 +90,5 @@ DEPENDENCIES
   rouge
   s3_website
 
-RUBY VERSION
-   ruby 2.3.1p112
```
Comment (shifi) [inline Gemfile.lock]: ?

Comment (thompiler) [inline Gemfile.lock]: Ah that snuck in by mistake. 😃

**Comment by thompiler** [comment]
PTAL

**Comment by thompiler** [comment]
>Thanks @thompiler

No prob! Thanks for reviewing.

**Pull request merged**